tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_paginationAndSortOrderMessages.js

```diff
@@ -43,25 +42,30 @@ describe('Order messages pagination', async () => {
     page = await helper.newTab(browser);
     this.pageObjects = await init();
   });
+
   after(async () => {
     await helper.closeBrowser(browser);
   });
+
   // Login into BO
   loginCommon.loginBO();
 
   it('should go to order messages page', async function () {
     await testContext.addContextItem(this, 'testIdentifier', 'goToOrderMessagesPage', baseContext);
-    await this.pageObjects.boBasePage.goToSubMenu(
-      this.pageObjects.boBasePage.customerServiceParentLink,
-      this.pageObjects.boBasePage.orderMessagesLink,
+
+    await this.pageObjects.dashboardPage.goToSubMenu(
+      this.pageObjects.dashboardPage.customerServiceParentLink,
+      this.pageObjects.dashboardPage.orderMessagesLink,
     );
-    await this.pageObjects.boBasePage.closeSfToolBar();
+
+    await this.pageObjects.dashboardPage.closeSfToolBar();
     const pageTitle = await this.pageObjects.orderMessagesPage.getPageTitle();
     await expect(pageTitle).to.contains(this.pageObjects.orderMessagesPage.pageTitle);
   });
 
   it('should reset all filters and get number of order messages in BO', async function () {
     await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
     numberOfOrderMessages = await this.pageObjects.orderMessagesPage.resetAndGetNumberOfLines();
     if (numberOfOrderMessages !== 0) await expect(numberOfOrderMessages).to.be.above(0);
```

Comment:
```suggestion
    await expect(numberOfOrderMessages).to.be.above(0);
```

Can you fix that, i think it's a bad copy past that i did

Reply:
I think it's not false because there is 0 order message by default

Reply:
after a fresh installation it's at least message.
![image](https://user-images.githubusercontent.com/48441421/81951294-48497d00-9605-11ea-9e19-425adecd6b2c.png)

Reply:
Done :)